### ISSUE TITLE
Crashing

### ISSUE THREAD

**Issue by Trashticz** [bebeli555/CookieClient#21]
Every time I launch the client using forge 1.12.2 It crashes right away
[crash-2021-09-06_18.10.06-client.txt](https://github.com/bebeli555/CookieClient/files/7118110/crash-2021-09-06_18.10.06-client.txt)

**Comment by bebeli555** [comment]
Thats an issue with baritone and to fix it you need to remove -noverify from ur jvm args

**Comment by kingofhenz** [comment]
Skill issue

**Comment by yang2266** [comment]
it crash went i launch at forger 1.12.2 
[crash-2022-11-26_15.48.09-client.txt](https://github.com/bebeli555/CookieClient/files/10095889/crash-2022-11-26_15.48.09-client.txt)
